src/components/DetailsNavBar/Evolution/Evolution.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import axios from "axios";
+import EvolutionChain from "./EvolutionChain.js";
+import { pokemonSpeciesRoute } from "../../../utils/baseUrl.js";
```

Comment:
Remove the js .. in imports .. also components are tsx and normal javascript files should be .ts